mod_ci/controllers.py

```diff
@@ -481,6 +491,15 @@ def queue_test(db, gh_commit, commit, test_type, branch="master", pr_nr=0):
             windows_test.platform.value: windows_test.id
         }
         for platform_name, test_id in status_entries.items():
+            if record_from_db is not None:
+                log.critical("Error. User in Blacklist!")
+                gh_commit.post(
+                   state=Status.ERROR,
+                   description="Tests cancelled! You may be blocked.",
+                   context="CI - {name}".format(name=platform_name),
+                   target_url=url_for('test.by_id', test_id=test_id, _external=True)
+                   )
+                return "Blocked User!"  # https://www.youtube.com/watch?v=AOHy4Ca9bkw
```

Comment:
The tests are left behind, and will be executed anyway next time another one is queued. That's not good.